terraspace/app/stacks/event/main.tf

```diff
@@ -42,6 +42,7 @@ module "event_delivery_lambda_from_sqs" {
     arn                                = aws_sqs_queue.event_delivery_queue.arn
     batch_size                         = var.batch_size
     maximum_batching_window_in_seconds = 30
+    function_response_types            = "ReportBatchItemFailures"
```

Comment:
That has to be a list. Check:
- https://github.com/hashicorp/terraform-provider-aws/issues/22068#issuecomment-999399168
- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_event_source_mapping

Reply:
thx. changed here https://github.com/elastic-ipfs/infrastructure/pull/156/commits/a05a0ad81d3c1f57dba1be4f3d8e267fc6020117